src/credentials/examples/ExampleDACs.cpp

```diff
@@ -29,7 +29,7 @@ namespace chip {
 namespace DevelopmentCerts {
 
 #if CHIP_DEVICE_CONFIG_DEVICE_VENDOR_ID == 0xFFF1 && CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_ID == 0x8000
-const uint8_t kDevelopmentDAC_Cert_FFF1_8000[493] = {
+const uint8_t kDevelopmentDAC_Cert_FFF1_8000[] = {
```

Comment:
Like the comment at the top of the file says, this is a generated file.  Please don't change it without changing the generator.

Reply:
Reverted the changes to this file...